aten/src/ATen/native/quantized/README.md

```diff
@@ -6,25 +6,26 @@ This document serves as an entry point for quantized kernel implementation.
 
 ## Implementing native quantized ops
 
-The new quantized ops are almost always located under the `ATen/native/quantized/cpu` folder.
-For the sake of an example, let us implement an element-wise quantized logical AND operation under `ATen/native/quantized/cpu/qand.cpp`.
+The new quantized ops are almost always located under the `ATen/native/quantized/cpu` folder. For
```

Comment:
why do you change `README` of implementing quantized kernels?

Reply:
Because the old example that implement `and` is outmoded by this PR. I replaced it with an example operator that does not exist after this PR